src/System.Collections.NonGeneric/tests/StackTests.cs

```diff
@@ -563,7 +563,7 @@ public void GetSyncRootBasic()
                 stackMother.Push(i);
             }
 
-            Assert.Equal(typeof(object), stackMother.SyncRoot.GetType());
+            Assert.Equal(typeof(Stack), stackMother.SyncRoot.GetType());
```

Comment:
Sorry not your fault but can you please change that with an Asser.IsType or whatever that method is called.